<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetFanShowForm</class>
 <widget class="QWidget" name="WidgetFanShowForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>84</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="color_widgets::ColorSelector" name="ColorSelector_1"/>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>RPM</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="color_widgets::ColorSelector" name="ColorSelector_2"/>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>Duty</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="color_widgets::ColorSelector" name="ColorSelector_3"/>
   </item>
   <item row="2" column="5">
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>Sensor</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QLabel" name="lDescription">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QLabel" name="lName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>color_widgets::ColorPreview</class>
   <extends>QWidget</extends>
   <header>color_preview.hpp</header>
  </customwidget>
  <customwidget>
   <class>color_widgets::ColorSelector</class>
   <extends>color_widgets::ColorPreview</extends>
   <header>color_selector.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
